LLama.Examples/ExampleRunner.cs

```diff
@@ -29,6 +29,8 @@ public class ExampleRunner
         { "Batched Executor: Fork", BatchedExecutorFork.Run },
         { "Batched Executor: Rewind", BatchedExecutorRewind.Run },
         { "Batched Executor: Guidance", BatchedExecutorGuidance.Run },
+        { "Whisper.net: Speech Recognition", SpeechTranscription.Run },
```

Comment:
It seems that this example uses nothing of LLamaSharp, though it's used in the other example as an util. Could you please move this part of code into the other example as a method?

Reply:
If I'm not misunderstanding it, the `ApeechChat` example already supports speech recognition internally. If that's right, it's totally okay to remove this example. But if you would like to, you could add a link of your own repo to tell other users to find the example of pure speech recognition to let others know your work. :)

Reply:
Great! I think the `SpeechChat` example is enough yup!